### PR TITLE
Transaction interpretation helpers

### PR DESCRIPTION
**Pull request by razvantomegea**

### CODE REVIEW

File: src/apiCalls/index.ts [hunk 1]
```diff
@@ -1,2 +1,6 @@
+export * from './account';
 export * from './configuration';
 export * from './endpoints';
+export * from './tokens';
```
Comment (arhtudormorar) [inline src/apiCalls/index.ts:4]: I think this was added ?

File: src/apiCalls/transactions/getTransactionsByHashes.ts [hunk 2]
```diff
@@ -14,7 +14,7 @@ export const getTransactionsByHashes = async (
   const { apiAddress } = networkSelector(getState());
   const hashes = pendingTransactions.map((tx) => tx.hash);
 
-  const { data: responseData } = await axios.get(
+  const { data: responseData } = await axiosInstance.get(
```
Comment (arhtudormorar) [inline src/apiCalls/transactions/getTransactionsByHashes.ts:17]: Why axiosInstance everywhere?

File: package.json [hunk 3]
```diff
@@ -65,7 +65,7 @@
   "devDependencies": {
     "@eslint/js": "9.15.0",
     "@multiversx/sdk-core": ">= 13.5.0",
-    "@multiversx/sdk-dapp-utils": "1.0.0",
```
Comment (arhtudormorar) [inline package.json:68]: `>=` & update peer dependency >= 1.0.2

File: src/apiCalls/index.ts (hunk 1 above)
Comment (razvantomegea) [reply src/apiCalls/index.ts:4]: Where?

File: src/apiCalls/configuration/getNetworkConfigFromApi.ts [hunk 4]
```diff
@@ -1,7 +1,7 @@
 import axios from 'axios';
 import { ApiNetworkConfigType } from 'types/network.types';
 import { NETWORK_CONFIG_ENDPOINT } from '../endpoints';
-import { getCleanApiAddress } from '../utils/getCleanApiAddress';
+import { getCleanApiAddress } from '../utils';
```
Comment (arhtudormorar) [inline src/apiCalls/configuration/getNetworkConfigFromApi.ts:4]: use from file

File: src/utils/transactions/getTransactionValue/getTransactionValue.ts [hunk 5]
```diff
@@ -0,0 +1,121 @@
...
+  getValueFromActions,
+  getValueFromDataField,
+  getValueFromOperations
+} from './helpers';
+import {
+  EgldValueDataType,
+  NFTValueDataType,
+  TokenValueDataType
+} from './types';
+import { InterpretedTransactionType } from '../../../types/serverTransactions.types';
```
Comment (arhtudormorar) [inline src/utils/transactions/getTransactionValue/getTransactionValue.ts:23]: check import

File: src/utils/transactions/getTransactionValue/getTransactionValue.ts [hunk 6]
```diff
@@ -0,0 +1,125 @@
...
+} from './helpers';
+import {
+  EgldValueDataType,
+  NFTValueDataType,
+  TokenValueDataType
+} from './types';
+import { InterpretedTransactionType } from 'types/serverTransactions.types';
+import { getTransactionActionNftText } from '../getTransactionActionNftText';
+import { getTransactionTokens } from '../getTransactionTokens';
+import { formatAmount } from '../../../lib/sdkDappUtils';
```
Comment (arhtudormorar) [inline src/utils/transactions/getTransactionValue/getTransactionValue.ts:26]: absolute?

**Review by arhtudormorar — APPROVED** [review]

**Pull request merged**